routes/defRoute.js

```diff
@@ -66,10 +66,25 @@ router.put(
   }
 );
 
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const defibrillator = await Defibrillator.findById(id);
+    return res.status(200).send({
+      error: false,
+      defibrillator
+    });
+  } catch (e) {
+    errorHandler(res, e);
```

Comment:
In the next pull request i will change this function.
So change it to - resServerError(res, e);